scipy/special/orthogonal.py

```diff
@@ -68,7 +68,7 @@
    *Fast computation of Gauss quadrature nodes and
    weights on the whole real line*.
    IMA Journal of Numerical Analysis
-   :doi:`10.1093/imanum/drv002`.
+   https://academic.oup.com/imajna/article-abstract/36/1/337/2363694
```

Comment:
Another redirect that I'm happy to remove if they don't beyond in this PR

Reply:
This seems undesired, DOIs are unique identifiers that are guaranteed to remain stable and can be easily looked up. We should prefer them to fixed links.

Reply:
Can you undo this and the next ones?

If it's broken, there might be an issue with Sphinx support for `:doi:`, but better to wait for a fix.

Reply:
The doi redirect from
https://docs.scipy.org/doc/scipy/reference/generated/scipy.special.roots_hermite.html
seems to work fine, so best left as is.

Reply:
reverted all the changes to DOI's